services/skus/controllers.go

```diff
@@ -943,54 +947,79 @@ func MerchantTransactions(service *Service) handlers.AppHandler {
 	})
 }
 
-func VerifyCredentialV2(service *Service) handlers.AppHandler {
+func handleVerifyCredV2(svc *Service, valid *validator.Validate) handlers.AppHandler {
 	return func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
 		ctx := r.Context()
 
-		l := logging.Logger(ctx, "skus").With().Str("func", "VerifyCredentialV2").Logger()
+		lg := logging.Logger(ctx, "skus").With().Str("func", "handleVerifyCredV2").Logger()
+
+		data, err := io.ReadAll(io.LimitReader(r.Body, reqBodyLimit10MB))
+		if err != nil {
+			lg.Warn().Err(err).Msg("failed to read body")
+
+			return handlers.WrapError(err, "Error in request body", http.StatusBadRequest)
+		}
+
+		req, err := parseVerifyCredRequestV2(data)
+		if err != nil {
+			lg.Warn().Err(err).Msg("failed to deserialize request")
```

Comment:
This text is used in at least 5 other places I've changed, and those passed previous reviews. You reviewed ]the fist PR](https://github.com/brave-intl/bat-go/pull/1916#discussion_r1287235021) that used this text, and since then there had been no questions.

This is consistent, and I don't see an issue with this. While the name of the function contains `parse`, the process involves a bit more than that, and it seemed appropriate to generalise and refer to it as deserialisation.

Thoughts?

Reply:
Yes, the link is correct – it shows the place where I had this text first.

> Arguably deserilaztion is more specific than parsing so to generalise it might be better the other way around.

Yes, agreed. I imprecisely expressed what I was hoping to say. Instead of `generalise`, it should have been `specify`. Because what's happening in the cases I used this message is deserialisation (i.e. parsing and constructing an object).

However, since the functions contain `parse` in their names it seems appropriate to log with `parse` as well.

Let me update it.